package.json

```diff
@@ -9,6 +9,8 @@
     "@playwright/test": "1.19.2",
     "allure-playwright": "2.0.0-beta.15",
     "babel-eslint": "10.1.0",
+    "babel-loader": "^8.2.3",
```

Comment:
let's pin these

Reply:
I think we can also get rid of cross-env, istanbul-instrumenter-loader, karma-coverage-istanbul-reporter, and more

Reply:
pinned dependencies

I think karma-coverage-istanbul-reporter is still needed for karma to read and report the coverage

Reply:
We can't remove cross-env because it is still needed for `NODE_OPTIONS`